TheaterDatabase/dbo/Tables/Performance.sql

```diff
@@ -0,0 +1,20 @@
+﻿CREATE TABLE [dbo].[Performance] (
+    [PerformanceId] INT identity(1,1) NOT NULL,
+    [Duration]      INT  NOT NULL,
+    [MinPrice]      INT  NOT NULL,
+    [MaxPrice]      INT  NOT NULL,
+    [MinimumAge]    INT  NOT NULL,
+    [Description]   TEXT NULL,
+    [MainImageId]   INT  NOT NULL,
+    [ProducerId]    INT  NULL,
+    [AuthorId]      INT  NULL,
+    [ArtistId]      INT  NULL,
+    [ComposerId]    INT  NULL,
+    CONSTRAINT [PK_PerformanceDetails] PRIMARY KEY CLUSTERED ([PerformanceId] ASC),
```

Comment:
Why PK_PerformanceDetails? 
Maybe just PK_Performance?

Reply:
Maybe, it'll be better, if MinPrice and MaxPrice will be a type of "decimal".